starboard/android/apk/app/src/main/java/dev/cobalt/media/VideoFrameReleaseTimeHelper.java

```diff
@@ -184,10 +196,11 @@ protected void onSynced() {
     // Do nothing.
   }
 
-  private boolean isDriftTooLarge(long frameTimeNs, long releaseTimeNs) {
+  private boolean isDriftTooLarge(long frameTimeNs, long releaseTimeNs, double playbackRate) {
     long elapsedFrameTimeNs = frameTimeNs - syncFramePresentationTimeNs;
     long elapsedReleaseTimeNs = releaseTimeNs - syncUnadjustedReleaseTimeNs;
-    return Math.abs(elapsedReleaseTimeNs - elapsedFrameTimeNs) > MAX_ALLOWED_DRIFT_NS;
+    return Math.abs(elapsedReleaseTimeNs - elapsedFrameTimeNs / playbackRate)
```

Comment:
Double-checking, should this be:
   ((elapsedReleaseTimeNs - elapsedFrameTimeNs) /  playbackRate) > MAX_ALLOWED_DRIFT_NS

Reply:
No, `elapsedReleaseTimeNs` is the difference between system time, and `elapsedFrameTimeNs` is the difference between frame presentation time.

Reply:
No, `elapsedReleaseTimeNs` is the difference between system time, and `elapsedFrameTimeNs` is the difference between frame presentation time.